connections/stream-migration.md

```diff
@@ -100,6 +133,21 @@ plantuml stream-migration.md -o stream-migration -tsvg
 
 Note: some of these steps may be pipelined.
 
+
+### Stream IDs
+
+In the above diagram stream IDs have the labels `A` and `B`. In practice this
+ID will be represented as an int defined by the initiator. Both sides can
+globally identify this stream by the namespacing the ID with the initiator's
```

Comment:
This would be an implementation detail, right? We wouldn't use this ID anywhere (on the wire).

Reply:
correct